src/compiler/parser.ts

```diff
@@ -2960,6 +2960,7 @@ namespace ts {
                 case SyntaxKind.UnknownKeyword:
                 case SyntaxKind.StringKeyword:
                 case SyntaxKind.NumberKeyword:
+                case SyntaxKind.BigIntKeyword:

```

Comment:
Need to handle bigint in `parsePropertyNameWorker`

Reply:
I don't think `bigint`s are valid property names, e.g. `{1n: 2}` throws a `SyntaxError` in V8